lib/config/presets/internal/default.ts

```diff
@@ -362,7 +362,7 @@ export const presets: Record<string, Preset> = {
     },
   },
   onlyNpm: {
-    description: 'Renovate only npm dependencies.',
+    description: 'Only update npm dependencies.',
```

Comment:
The description implies that Renovate will only update npm dependencies, but the config disables `docker-compose`, `dockerfile` and `meteor`.

I must be missing something here. In any case, the current `description` seems misleading to me.

Maintainer input wanted. 😉 

> ```
>   onlyNpm: {
>     description: 'Only update npm dependencies.',
>     'docker-compose': {
>       enabled: false,
>     },
>     dockerfile: {
>       enabled: false,
>     },
>     meteor: {
>       enabled: false,
>     },
>   },
> ```

Reply:
Ha, this is a very old preset! I think maybe time to remove it seeing as it clearly doesn't work anymore.

Reply:
@rarkins want me to remove the `onlyNpm` preset then?

Reply:
Yes, I think we have a list of removed presets which it should be added to, that way users don't get _errors_ if they still include it

Reply:
Is this the list you're thiking of?

https://github.com/renovatebot/renovate/blob/d10dcff495c7de06f324e0d26e6add6ee08747d6/lib/config/presets/common.ts#L1-L31